### PR TITLE
added readme and printed output to console

### PR DESCRIPTION
**Pull request by alankar63**

### CODE REVIEW

File: sorted_rating.sh [hunk 1]
```diff
@@ -28,16 +18,15 @@ for f in $dir/*
    name=$(echo ${orig_name//./+})
    name=$(echo ${name//_/+})
    name=$(echo ${name// /+})
-   curl -s  http://www.imdb.com/find\?ref_\=nv_sr_fn\&q\=$name\&s\=all | \
-   grep -E -0  '/tt\w+' |cut -c72- > out.txt
+   curl -s  http://www.imdb.com/find\?ref_\=nv_sr_fn\&q\=$name\&s\=all \
+   |grep -E -0  '/tt\w+' |cut -c72- > out.txt
```
Comment (vaidik) [inline sorted_rating.sh]: Indendtation required here. Check the bash guidelines on how to do it.

File: sorted_rating.sh [hunk 2]
```diff
@@ -46,3 +35,4 @@ for f in $dir/*
  done
 
 sort final_rate.txt -o final_rate.txt
+less final_rate.txt
```
Comment (vaidik) [inline sorted_rating.sh]: Great. Now can you get rid of the file as well? You don't need files for this .

**Comment by vaidik** [comment]
This entire PR is using files. Try to get rid of files. Use variables and pipes and that should be enough.

File: sorted_rating.sh [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-2.1. If best match is a future movie than it is kept as lowest rating movie
-2.2. Movies with same rating are sorted with name 
-3.   assumptions :
-3.1  directory to be provided after bash command
-4.   Output file :final_rate.txt
----------------------------------------------------------------------------
-'
+#!/bin/bash
 
 touch final_rate.txt
```
Comment (vishesh92) [inline sorted_rating.sh]: I don't think you need this.

File: sorted_rating.sh [hunk 4]
```diff
@@ -28,16 +18,15 @@ for f in $dir/*
...
+   |grep -E -0  '/tt\w+' |cut -c72- > out.txt
    #movie id identified
    
    cut -c-9 out.txt > final.txt
    read val < final.txt
    curl -s http://www.imdb.com/title/$val/ > rating.html
    data=$(grep "based on" rating.html| grep "[0-9]\.[0-9]" -o)
-   data1=${data:0:3}
-   #rating kept in data
+   data1=${data:0:3}    #rating kept in data
```
Comment (vishesh92) [inline sorted_rating.sh]: Leave a space after `#`.

**Pull request merged**